### PR TITLE
Fixes #14451: update mustache GM documentation to detail the windows syntax

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/14451

### CODE REVIEW

File: tree/30_generic_methods/file_from_template_mustache.cf [hunk 1]
```diff
@@ -69,8 +69,9 @@
 #
 # * Use `{{#vars.container}} content {{/vars.container}}` to iterate
 # * Use `{{{.}}}` for the current element value in iteration
-# * Use `{{{.key}}}` for the `key` value in current element
-# * Use `{{{@}}}` for the current element key in iteration
+# * Use `{{{.key}}}` for the `key` value in current element on Linux nodes
```
Comment (amousset) [inline tree/30_generic_methods/file_from_template_mustache.cf]: I think the syntax without a dot works everywhere

**Comment by Fdall** [comment]
Commit modified

File: tree/30_generic_methods/file_from_template_mustache.cf [hunk 2]
```diff
@@ -69,8 +69,8 @@
 #
 # * Use `{{#vars.container}} content {{/vars.container}}` to iterate
 # * Use `{{{.}}}` for the current element value in iteration
-# * Use `{{{.key}}}` for the `key` value in current element on Linux nodes
-# * Use `{{{key}}}` for the `key` value in current element on Windows nodes
+# * Use `{{{key}}}` for the `key` value in current element on Windows and Linux nodes
```
Comment (amousset) [inline tree/30_generic_methods/file_from_template_mustache.cf]: no need to specify the OS then

**Comment by Fdall** [comment]
Commit modified

**Pull request merged**